model_tag_def.go

```diff
@@ -8,25 +8,23 @@
  */
 
 package schemaregistry
-import (
-	"time"
-)
+
 // TagDef struct for TagDef
 type TagDef struct {
-	Category string `json:"category,omitempty"`
-	Guid string `json:"guid,omitempty"`
-	CreatedBy string `json:"createdBy,omitempty"`
-	UpdatedBy string `json:"updatedBy,omitempty"`
-	CreateTime time.Time `json:"createTime,omitempty"`
-	UpdateTime time.Time `json:"updateTime,omitempty"`
-	Version int64 `json:"version,omitempty"`
-	Name string `json:"name,omitempty"`
-	Description string `json:"description,omitempty"`
-	TypeVersion string `json:"typeVersion,omitempty"`
-	ServiceType string `json:"serviceType,omitempty"`
-	Options map[string]string `json:"options,omitempty"`
+	Category      string              `json:"category,omitempty"`
+	Guid          string              `json:"guid,omitempty"`
+	CreatedBy     string              `json:"createdBy,omitempty"`
+	UpdatedBy     string              `json:"updatedBy,omitempty"`
+	CreateTime    int64               `json:"createTime,omitempty"`
```

Comment:
What's the reason for this? It seems less intuitive than before.

Reply:
The Stream Catalog API returns epoch time format for `createTime` and `updateTime` fields. This change is already made in the API spec (https://github.com/confluentinc/api/pull/910). This ticket (https://confluentinc.atlassian.net/browse/DGS-5954) was created to update the SDK to support the appropriate field type.